PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx

```diff
@@ -240,6 +242,12 @@ Bool_t AliAnalysisTaskNewJetSubstructure::Run() {
   // Run analysis code here, if needed. It will be executed before
   // FillHistograms().
 
+  if (fRejectTPCPileup)   {            //possible bug occurs here, use with caution
```

Comment:
Is this comment correct? (via grep, I think it's from the Jet Extractor?) . If obsolete, I would encourage you to remove it